docs/references/keywords.rst

```diff
@@ -186,7 +186,7 @@ extensions).
         ``data_files`` is deprecated. It does not work with wheels, so it
         should be avoided.
 
-    A list of strings specifying the data files to install.
+    A sequence of (*directory*, *files*) pairs.
```

Comment:
Should we also say that `files` is a list of files itself?

Reply:
@abravalheri yep I've expanding on what exactly data_files should take

Reply:
Thank you very much.